packages/eslint-plugin-office-addins/src/rules/office-ready.ts

```diff
@@ -0,0 +1,30 @@
+import { TSESTree } from "@typescript-eslint/typescript-estree";
+
+export = {
+  name: "office-ready",
+  meta: {
```

Comment:
Take a look at the preferred rule naming conventions from eslint.  Should probably be something like "use-office-on-ready" or "no-office-initialize"

Reply:
I followed this convention: [Rules Convention](https://eslint.org/docs/developer-guide/working-with-rules#rule-naming-conventions). It doesn't explicit say what to do in case of a "positive rule".
In this case, I do think the "no-office-initialize" could be proper and I will be using that now